### ISSUE TITLE
Add a11y attributes to the Input

### ISSUE THREAD

**Issue by kentcdodds** [paypal/react-autocompletely#39]
Based on `react-autocomplete` in [this example](https://reactcommunity.org/react-autocomplete/managed-menu-visibility/) here's what the input renders:

```javascript
<input id="states" role="combobox" aria-autocomplete="list" aria-expanded="true" autocomplete="off" value="">
```

We should probably add those to ours I think :smile:

Should be pretty straightforward for anyone familiar with react.

**Comment by salmanmanekia** [comment]
I will take this. If I understand this correctly then only the aria-expanded should be either true or false based on if the onClick event is triggered on the input element. Other elements are 
always
 ```
        role="combobox"
        aria-autocomplete="list"
        autocomplete="off"
```

**Comment by kentcdodds** [comment]
> If I understand this correctly then only the aria-expanded should be either true or false based on if the onClick event is triggered on the input element. Other elements are
always

This is mostly correct. For the `aria-expanded` one, you should just be able to get the `isOpen` state from `this.autocomplete.state.isOpen`.

They should also all be overrideable (so the `...rest` should appear _after_ these).
